.github/actions/release/changelog-generator/action.yml

```diff
@@ -43,13 +59,28 @@ runs:
           --project ${{ env.GITHUB_REPOSITORY_PROJECT }} 
           --token ${{ inputs.github-token }} 
           --future-release ${{ inputs.future-release }} 
+          ${{ env.SINCE_ARG }}
           ${{ inputs.optional-arg }}
 
     - name: Populate changelog descriptions
       shell: bash
       run: |
         .github/workflows/scripts/populate-changelog-descriptions.sh ${{ inputs.changelog-path }}
 
+    - name: Cut other releases
+      if: ${{ inputs.output-only-current-release }}
+      shell: bash
+      run: |
+        ESCAPED_CURRENT_VERSION=${${{ inputs.previous-release }}//./\\.}
+        # cut everything before the current release changes
+        sed -n -z -E "s/.*$ESCAPED_CURRENT_VERSION\)(.*)/\1/p" -i ${{ inputs.changelog-config-path }}
```

Comment:
This also removes the `Full Changelog` link to the commits added after the last version, is that intentional?

Reply:
Yes, the thought was, that the github releases page contains all the removed information in one way or the other.

Reply:
Fair point, I think removing it is a-okay then.